redox/response_flowsheet.go

```diff
@@ -341,10 +352,15 @@ func formatInt(val *int) string {
 }
 
 func formatFloat(val *float64) string {
+	return formatFloatWithPrecision(val, 4)
+}
+
+func formatFloatWithPrecision(val *float64, decimalPlaces int) string {
 	if val == nil {
 		return missingValue
 	}
-	return fmt.Sprintf("%.4f", *val)
+	tpl := fmt.Sprintf("%%.%df", decimalPlaces)
+	return fmt.Sprintf(tpl, *val)
```

Comment:
```suggestion
	return fmt.Sprintf("%%.*f", decimalPlaces, *val)
```
> Either or both of the flags may be replaced with the character '*', causing their values to be obtained from the next operand (preceding the one to format), which must be of type int.

Reply:
Your suggestion didn't work. I'm reverting the commit.

Reply:
Ugh, it should have only been a single %: https://go.dev/play/p/8r9U7Er1-Iy